adam-core/src/main/scala/org/bdgenomics/adam/util/Util.scala

```diff
@@ -16,9 +16,13 @@
  * limitations under the License.
  */
 package org.bdgenomics.adam.util
-
 import org.bdgenomics.formats.avro.Contig
 
+// We may want to reintroduce in the future MD5 concordance test for isSameContig as
```

Comment:
I don't like to leave dead code in. Let's either remove the whole comment + code, or keep the comment but include a commit hash for when the code was removed so that it is easy to track down the diff with the old code.

As an aside, I would remove this class. IMO, it's not very useful to have a function that just calls `String.equals`.